core/client-internal/src/test/java/tachyon/client/block/TachyonBlockStoreTest.java

```diff
@@ -97,7 +96,6 @@
   public void before() throws Exception {
     mTestFile = mTestFolder.newFile("testFile");
 
-    ClientContext.reset();
```

Comment:
The powermock modification of `BlockStoreContext.INSTANCE` is local to the test class, but it is messy to not reset it between tests in the same file. I've moved most functionality to `@BeforeClass`